query/branch/default_branch_requires_2_pull_request_reviews.sql

```diff
@@ -0,0 +1,13 @@
+select
+  -- Required Columns
+  r.full_name as resource,
+  case
+    when b.required_pull_request_reviews is not null and (b.required_pull_request_reviews -> 'required_approving_review_count')::integer >= 2 then 'ok'
+    else 'alarm'
+  end as status,
+    r.full_name || ' default branch ' || r.default_branch || case when(b.required_pull_request_reviews is not null and (b.required_pull_request_reviews -> 'required_approving_review_count')::integer >= 2) then ' has ' else ' does not have ' end || 'at least two pull request reviews.' as reason,
```

Comment:
can we update the format as per - https://github.com/turbot/steampipe-mod-github-compliance/pull/4/files#r946451106